app/controllers/api_controller.rb

```diff
@@ -213,7 +213,7 @@ def supported_relationships
   end
 
   def api_version
-    res = request.headers['Accept'].match(REGEXP_API_VERSION)
+    res = request.headers['Accept']&.match(REGEXP_API_VERSION)
```

Comment:
Support missing Accept headers in requests. This method will return nil in this case.